DQM/SiStripMonitorDigi/src/SiStripMonitorDigi.cc

```diff
@@ -291,23 +294,23 @@ void SiStripMonitorDigi::endLuminosityBlock(const edm::LuminosityBlock& lb, cons
     int nFeds [6] = { 96 , 96 , 84 , 15 , 15 , 134 }; // tec- , tec+ , tib , tid- , tid+ , tob
     int nFedsConnected [6] = { nFedTECm , nFedTECp , nFedTIB , nFedTIDm , nFedTIDp , nFedTOB };
 
-    MonitorElement * me = dqmStore_->get(topFolderName_+"/MechanicalView/NumberOfDigisInLastLS");     
```

Comment:
@dmitrijus ok, I have corrected the above. There was really no need for this line anyway as the MonitorElement it is getting had already been instantiated elsewhere in the code and is still in scope as far as I can see.